[2021-08-26 01:45:04,925] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test_dag.t1 2021-08-26T01:45:04.290653+00:00 [queued]>
[2021-08-26 01:45:04,936] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test_dag.t1 2021-08-26T01:45:04.290653+00:00 [queued]>
[2021-08-26 01:45:04,937] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-26 01:45:04,938] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-26 01:45:04,941] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-26 01:45:04,955] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): t1> on 2021-08-26T01:45:04.290653+00:00
[2021-08-26 01:45:04,962] {standard_task_runner.py:52} INFO - Started process 6671 to run task
[2021-08-26 01:45:04,966] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 't1', '2021-08-26T01:45:04.290653+00:00', '--job-id', '27', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpl5_aiwhl', '--error-file', '/tmp/tmpi7z1z12q']
[2021-08-26 01:45:04,968] {standard_task_runner.py:77} INFO - Job 27: Subtask t1
[2021-08-26 01:45:05,017] {logging_mixin.py:104} INFO - Running <TaskInstance: test_dag.t1 2021-08-26T01:45:04.290653+00:00 [running]> on host f913d28eb98c
[2021-08-26 01:45:05,078] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dini
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-08-26T01:45:04.290653+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-26T01:45:04.290653+00:00
[2021-08-26 01:45:05,103] {python.py:151} INFO - Done. Returned value was: None
[2021-08-26 01:45:05,114] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=t1, execution_date=20210826T014504, start_date=20210826T014504, end_date=20210826T014505
[2021-08-26 01:45:05,144] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-26 01:45:05,179] {local_task_job.py:151} INFO - Task exited with return code 0
